java/dagger/hilt/android/plugin/src/main/kotlin/dagger/hilt/android/plugin/HiltGradlePlugin.kt

```diff
@@ -34,6 +36,28 @@ class HiltGradlePlugin : Plugin<Project> {
     val androidExtension = project.extensions.findByType(BaseExtension::class.java)
       ?: throw error("The Hilt Android Gradle plugin can only be applied to an Android project.")
     androidExtension.registerTransform(AndroidEntryPointTransform())
+    val extension = project.extensions.create(
+      HiltExtension::class.java, "hilt", HiltExtensionImpl::class.java
+    )
+    val appExtension = project.extensions.findByType(AppExtension::class.java)
```

Comment:
You can fetch `TestedExtension`, and there is `TestedExtension.unitTestVariants` property.